modules/efk-logging-curator-configuration.adoc

```diff
@@ -3,30 +3,113 @@
+<5> Optionally, specify a file path to your CA certificate, in single quotes, to validate the SSL certificate used by Elasticsearch or leave empty.
+<6> Optionally, specify a file path to an SSL client cert file to authenticate to Elasticsearch. The file may contain both an SSL client certificate and an SSL key, in which case client_key is not used. If specifying client_cert, and the file specified does not also contain the key, use client_key to specify the file containing the SSL key. The file must be in PEM format, and the key part, if used, must be an unencrypted key in PEM format as well. Leave empty if not using a SSL client cert.
+<7> Optionally, specify an SSL client key file to authenticate to Elasticsearch. If using client_cert and the file specified does not also contain the key, use client_key to specify the file containing the SSL key. The key file must be an unencrypted key in PEM format. Leave empty if not using a SSL client cert.
+<8> Set to `True` to disable SSL certificate verification.
++
+[NOTE]
+====
+Setting ssl_no_validate to True will likely result in a warning message that your SSL certificates are not trusted. This is expected behavior.
+====
++
+<9> Optionally, specify authentication credentials, such as `user:pass`, or leave empty. 
+<10> Optionally, specify the default client connection timeout value, by default 30 seconds.
+<11> Optionally, set to `True` to run Curator on the master node only.
++
+For more information on these parameters, see the link:https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/configfile.html[Elastic Search curator documentation].
+
+. Set the minimum acceptable log severity to display.
 +
 ----
-oc create job --from=cronjob/logging-curator <job_name>
+logging:
+  loglevel: INFO <1>
+  logfile:
+  logformat: default
+  blacklist: ['elasticsearch', 'urllib3']
 ----
 +
+Specify the log level:
++
+* *CRITICAL*. Curator displays only critical messages.
+* *ERROR*. Curator displays only  error and critical messages.
+* *WARNING*. Curator displays only  error, warning, and critical messages.
+* *INFO*. Curator displays only informational, error, warning, and critical messages.
+* *DEBUG*. Curator displays only debug messages, in addition to all of the above. 
++
+The default value is INFO.
+
+[NOTE]
+====
+Cluster Logging uses the {product-title} custom environment variable `CURATOR_SCRIPT_LOG_LEVEL` in {product-title} wrapper scripts (run.sh and convert.py).
+The environment variable takes the same values as `CURATOR_LOG_LEVEL` for script debugging, as needed.
+====
+
+[[efk-logging-curator-configuration-cronjob]]
+== Configuring Curator using a cronjob
+
+You can alternatively create configure and run Curator using a cronjob. The next scheduled job uses this configuration.
+
+[source,bash]
+----
+oc create job --from=cronjob/logging-curator <job_name>
```

Comment:
This is purely for debugging. Let's say admin notices that curator is in failed state, but they don't know why. So next logical thing to do would be to bump up the log level and run it again. And this command is what they would use, instead of waiting for 24 hours for another scheduled run of the cron job.